solidity/dashboard/src/css/liquidity-page.less

```diff
@@ -43,16 +77,65 @@
       display: flex;
       flex-direction: column;
       justify-content: center;
+      position: relative;
+
+      .liquidity__info-tile__tooltip {
+        width: 12px;
+        height: 12px;
+        position: absolute;
+        top: 0px;
+        right: 8px;
+        cursor: pointer;
+
+        .tooltip__content-wrapper {
+          text-align: left;
+          min-width: 16rem;
+        }
+      }
+    }
+  }
+
+  .liquidity__new-user-info {
+    background-color: @color-brand-violet;
+    color: @white;
+    display: flex;
+    padding: 0.7rem 0.5rem;
+
+    .liquidity__rewards-icon {
+      width: 2rem;
+      height: 2rem;
+      margin: 0 0.5rem;
+
+      path {
+        fill: white;
+      }
+    }
+
+    .liquidity__new-user-info-text {
+      margin: 0.2rem 4rem 0.2rem 0.5rem;
+      font-weight: 500;
+      line-height: 1rem;
+
+      span {
+        font-weight: 400;
+        font-size: 0.75rem;
+      }
```

Comment:
If we use `Banner`'s components, we probably can remove these styles. ref https://github.com/keep-network/keep-core/pull/2250#discussion_r551339330.

Reply:
I think we still need some of them, but i deleted the unnecessary ones.